assets/scripts/evolution/svg_pagination.js

```diff
@@ -0,0 +1,16 @@
+export default function() {
+	(function() {
+		var elems = document.querySelectorAll('ul.evo_c-svg-dot--default li'),
+		makeActive;
+
+		makeActive = function() {
```

Comment:
@yyrcat 

This code will work perfectly when `ul.evo_c-svg-dot--default li` exists in the markup. In cases where those elements are not present, the code will break.. The solution here is to check if `elems` exist. See detailed instructions here on writing safe javascript code: https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#writing-safe-javascript-code